apps/api/docs/development-setup.md

```diff
@@ -81,7 +81,7 @@ Make sure Redis and MariaDB server are up and running.
    TWILIO_WA_NUMBER=
 
    # TEMP
-   APP_NAME=osmo_notify
+   APP_NAME=osmoX
```

Comment:
Make it `OsmoX`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b